src/main/java/com/codegile/univ/mihai/week1/day1/Project1.java

```diff
@@ -0,0 +1,53 @@
+package com.codegile.univ.mihai.week1.day1;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+
+public class Project1 {
+
+    /*public void ReadFile(String txt){
+        InputStream inputStream = this.getClass().getResourceAsStream("text.txt");
+        new Scanner(inputStream);
+        System.out.println(inputStream);
+    }*/
+
+    public static void main(String args[]) throws IOException {
+        int nrReale = 0;
+        int nrIntregi = 0;
+
+        File file = new File("resources/day1/text.txt");
+        Scanner scanner = new Scanner(file);
+        String lineSeparator = System.getProperty("line.separator");
+
+
+
+        try {
+            while(scanner.hasNextLine()) {
+                    if(scanner.hasNextInt()){
+                        nrIntregi++;
+                        System.out.println(scanner.nextInt());
```

Comment:
the problem didn't ask for you to print the numbers or anything else apart from the count - usually it's best to limit solutions to the requirements. Remember the 'I want to move' 'I want to move and take my stuff' etc. excercise